src/CLICommand/parameters_helper.ts

```diff
@@ -323,3 +354,6 @@ export class ParametersHelper {
 		return !!dryRun;
 	}
 }
+
+const invalidSchemaError =
```

Comment:
Consider renaming to `invalidSchemaErrorMessage` or similar

Reply:
Renamed in Core: 080651c (Core)